libs/blocks/ost/ost.js

```diff
@@ -0,0 +1,48 @@
+import { loadScript, loadStyle } from '../../utils/utils.js';
+
+const IMS_COMMERCE_CLIENT_ID = 'aos_milo_commerce';
+const IMS_PROD_URL = 'https://auth.services.adobe.com/imslib/imslib.min.js';
+const OST_SCRIPT_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/index.js';
+const OST_STYLE_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/index.css';
+
+const getImsToken = async () => {
+  window.adobeid = {
+    client_id: IMS_COMMERCE_CLIENT_ID,
+    environment: 'stg1',
+    scope: 'AdobeID,openid',
```

Comment:
no need for adobeio scope?

Reply:
Is the exact value of the scope `adobeio`? I only see `adobeio_api`, `adobeio.appregistry.read` and `adobeio.appregistry.read` in IMS portal. All 3 options require business justification and approval, will wait for confirmation before submitting.
@yesil @honstar

Reply:
I'd say omit the `adobeio` scope if it is not needed for OST to function properly. I can't imagine why `adobeio` would be needed here in the first place, sounds totally unrelated.